CHANGES.rst

```diff
@@ -19,6 +19,8 @@
 
 - Fix waveband option in registry.regsearch [#175]
 
+- Move auth subpackage to permanent location [#178]
```

Comment:
bike shedding (but that is all I can offer here): I don't think this needs a changelog entry as it changes something that has never been released to the wild, so the users wouldn't know the location was something else originally.

Reply:
Yeah I'm with you on this one, I'd take it out, and just mark this pr as no-changelist-required

Reply:
Good point. Reverted.